tests/AutoDocControllerTest.php

```diff
@@ -12,18 +12,24 @@ class AutoDocControllerTest extends TestCase
     use PHPMock;
 
     protected static array $documentation;
-    protected static string $localDriverFilePath;
+    protected static string $baseFile;
 
     public function setUp(): void
     {
         parent::setUp();
 
-        self::$localDriverFilePath ??= __DIR__ . '/../storage/documentation.json';
+        $documentationDirectory = config('auto-doc.drivers.local.directory');
```

Comment:
as discussed, no need to get directory from the config as you already know it https://github.com/RonasIT/laravel-swagger/pull/152/files#diff-b75c65b4a559e813643f28660af83707342794a8f8b5d49c70fd8d3689d29614R131